### PR TITLE
Add ACLK Connection Details

### PR DESCRIPTION
**Pull request by zack-shoylev**
##### Summary
Make IPs are documented as well as broad types of data currently uploaded (to be made more specific going forward)
##### Component Name
doc update

##### Test Plan
none

##### Additional Information
https://github.com/netdata/marketing/issues/233

### CODE REVIEW

File: aclk/README.md [hunk 1]
```diff
@@ -11,6 +11,13 @@ The Agent-Cloud link (ACLK) is the mechanism responsible for securely connecting
 through Netdata Cloud. The ACLK establishes an outgoing secure WebSocket (WSS) connection to Netdata Cloud on port
 `443`. The ACLK is encrypted, safe, and _is only established if you claim your node_.
 
+The Cloud IP address is 35.196.244.138. In the future the IP ranges would be the ones documented [here](https://www.cloudflare.com/ips/).
```
Comment (joelhans) [inline aclk/README.md:14]:
```suggestion
The Cloud IP address is 35.196.244.138. In the future the IP ranges would be the ones [documented by Cloudflare](https://www.cloudflare.com/ips/).
```

File: aclk/README.md [hunk 2]
```diff
@@ -110,3 +117,6 @@ If you changed the runtime setting in your `var/lib/netdata/cloud.d/cloud.conf`
 Restart your Agent and [claim your node](/claim/README.md#how-to-claim-a-node).
 
 [![analytics](https://www.google-analytics.com/collect?v=1&aip=1&t=pageview&_s=1&ds=github&dr=https%3A%2F%2Fgithub.com%2Fnetdata%2Fnetdata&dl=https%3A%2F%2Fmy-netdata.io%2Fgithub%2Faclk%2FREADME&_u=MAC~&cid=5792dfd7-8dc4-476b-af31-da2fdb9f93d2&tid=UA-64295674-3)](<>)
+
+### ACLK Connection Details
```
Comment (joelhans) [inline aclk/README.md:121]:
```suggestion
### ACLK connection details
```

Comment (joelhans) [reply aclk/README.md:121]: This heading is currently at the bottom of the page. I feel like it should go somewhere else, unless there's some other plan for it?

**Comment by zack-shoylev** [comment]
@prologic this has been discussed. We will update this when we make that change.

File: aclk/README.md [hunk 3]
```diff
@@ -11,6 +11,13 @@ The Agent-Cloud link (ACLK) is the mechanism responsible for securely connecting
 through Netdata Cloud. The ACLK establishes an outgoing secure WebSocket (WSS) connection to Netdata Cloud on port
 `443`. The ACLK is encrypted, safe, and _is only established if you claim your node_.
 
+The Cloud IP address is 35.196.244.138.
+
+Netdata is a distributed monitoring system. Very few data are uploaded to the cloud, such as data about:
```
Comment (rduffield) [inline aclk/README.md:16]: I prefer "streamed" to "uploaded".

**Pull request merged**